src/render/index.ts

```diff
@@ -808,6 +809,7 @@ export const visualElement = <Instance, MutableState, Options>({
          * needs to be performed.
          */
         notifyLayoutReady(config) {
+            setCurrentViewportBox(element)
```

Comment:
Ah, did it move to here?

Reply:
Yeah that's right. Specifically what this changes is we now only run this when the element's layout is definietly ready to be used in projection calculations.